doc/source/examining/loading_data.rst

```diff
@@ -2030,6 +2038,35 @@ Note: for backward compatibility, particles from the
 default (including dark matter, stars, tracer particles, …). Sink
 particles have the particle type ``sink``.
 
+Particle automatic filtering
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If your RAMSES version is more recent than `stable_17_09`, it is
+possible to tell yt to filter the particles in your dataset. This is
+not done by default is it requires to read all the particles and may
+be small. To use this feature, run
```

Comment:
Not sure what the last sentence means. Maybe "This is not done by default as it requires that all of the particles must be read in from disk and this might be expensive on large datasets"?

Reply:
You're right, the sentence doesn't make sense. I'll reword it.